core/domain/stats_services_test.py

```diff
@@ -579,192 +817,161 @@ def _create_and_update_fake_exploration(self, exp_id):
+                            }, {
+                                'frequency': 1,
+                                'answer': 'a1',
+                                'state': self.STATE_2_NAME
+                            }, {
+                                'frequency': 1,
+                                'answer': 'a2',
+                                'state': self.STATE_2_NAME
+                            }]
+                        }
+                    })
 
     def test_unresolved_answers_count_for_non_default_rules(self):
-        exp_1 = self._create_and_update_fake_exploration('eid1')
-        self.assertEquals(
-            stats_services.get_exps_unresolved_answers_for_default_rule(
-                ['eid1']), self._get_default_dict_when_no_unresolved_answers(
-                    ['eid1']))
-        event_services.AnswerSubmissionEventHandler.record(
-            'eid1', 1, exp_1.init_state_name, self.CLASSIFIER_RULESPEC_STR, 'a1'
-        )
-        event_services.AnswerSubmissionEventHandler.record(
-            'eid1', 1, self.STATE_2_NAME, self.CLASSIFIER_RULESPEC_STR, 'a1')
-        self.assertEquals(
-            stats_services.get_exps_unresolved_answers_for_default_rule(
-                ['eid1']), self._get_default_dict_when_no_unresolved_answers(
-                    ['eid1']))
-
-
-class EventLogEntryTests(test_utils.GenericTestBase):
-    """Test for the event log creation."""
-
-    def test_create_events(self):
-        """Basic test that makes sure there are no exceptions thrown."""
-        event_services.StartExplorationEventHandler.record(
-            'eid', 2, 'state', 'session', {}, feconf.PLAY_TYPE_NORMAL)
-        event_services.MaybeLeaveExplorationEventHandler.record(
-            'eid', 2, 'state', 'session', 27.2, {}, feconf.PLAY_TYPE_NORMAL)
+        with self.swap(
+            jobs_registry, 'ALL_CONTINUOUS_COMPUTATION_MANAGERS',
+            self.ALL_CC_MANAGERS_FOR_TESTS):
+            exp_1 = self._create_and_update_fake_exploration('eid1')
+            self.assertEqual(
+                stats_services.get_exps_unresolved_answers_for_default_rule(
+                    ['eid1']),
+                self._get_default_dict_when_no_unresolved_answers(['eid1']))
+            self._record_answer(
+                'a1', exp_1, exp_1.init_state_name,
+                classification=exp_domain.STATISTICAL_CLASSIFICATION)
+            self._record_answer(
+                'a1', exp_1, self.STATE_2_NAME,
+                classification=exp_domain.STATISTICAL_CLASSIFICATION)
+            self._run_aggregator_job()
+            self.assertEqual(
+                stats_services.get_exps_unresolved_answers_for_default_rule(
+                    ['eid1']),
+                self._get_default_dict_when_no_unresolved_answers(['eid1']))
+
+
+class ExplorationStatsTests(test_utils.GenericTestBase):
+    """Tests for stats functions related to explorations."""
+    pass
```

Comment:
No clue, removed. It was likely part of a reminder for me to add tests for ``stats_services``, but I must have implemented them elsewhere in this file.